limitador-server/Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = "Apache-2.0"
 keywords = ["rate-limiting", "rate", "limiter", "envoy", "rls"]
 categories = ["web-programming"]
 description = "Rate limiting service that integrates with Envoy's RLS protocol"
-homepage = "https://github.com/kuadrant/limitador"
+homepage = "https://kuadrant.io"
 repository = "https://github.com/kuadrant/limitador"
 documentation = "https://docs.rs/limitador"
```

Comment:
I'd probably get rid of the `documentation` here, or have it point to [kuadrant.io/docs/limitador-server](https://kuadrant.io/docs/limitador-server/)? (There will be better content there soon™)